### PR TITLE
Added Zilliz to the list

### LINKED ISSUE #831: Add Zilliz to Hacktoberfest Swag List
[Zilliz](https://zilliz.com/hacktoberfest) is participating in Hacktoberfest 2023 with their Milvus project!

This issue is open for anyone who wants to contribute to this repo.

Please read the [contributing guideline](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) to understand the formatting and the information expected from the announcement for your PR to be successfully merged.

Please check the [active PRs](https://github.com/crweiner/hacktoberfest-swag-list/pulls) to make sure there isn't already an active PR for this issue.

### PR DESCRIPTION
**Pull request by bharatr21**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Closes #831 

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -673,6 +673,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
 
+### Z
+
+#### **Zilliz**
+
+- **Swag**: Swag Pack (t-shirts + stickers)
+- **Requirements**: 3 merged PRs in the Milvus project.
+- **How to sign up**: No special signup known.
+- **Issues**: See the list of open issues [here](https://github.com/milvus-io/milvus/issues?q=is%3Aopen+is%3Aissue+label%3AHacktoberfest).
+- **Notes**: Check out [Zilliz Hacktoberfest page](https://zilliz.com/hacktoberfest) and the [Contribution Guide](https://github.com/milvus-io/milvus/blob/5ba158a3f559e424e8bd98f3c2032d53f8cf310c/CONTRIBUTING.md) for more details. Additionally, you can follow them on Twitter [here](https://twitter.com/zilliz_universe?s=21&t=N_U76Etz0LitZhkwlsp6qg) and join the Discord [here](https://discord.gg/vMaKFtEU).
```
Comment (vimode) [inline docs/list.md:684]:
```suggestion
- **Notes**: Check out [Zilliz Hacktoberfest page](https://zilliz.com/hacktoberfest) and the [Contributing Guide](https://github.com/milvus-io/milvus/blob/5ba158a3f559e424e8bd98f3c2032d53f8cf310c/CONTRIBUTING.md) for more details.
```

**Review by vimode — CHANGES_REQUESTED** [review]
Hey @bharatr21 
The Zilliz Hacktoberfest page also mentions a few other ways to contribute, please add those in the requirements, as a way to encourage low/no-code contributions too.

File: docs/list.md (hunk 1 above)
Comment (bharatr21) [reply docs/list.md]: Updated this

**Comment by bharatr21** [comment]
@vimode updated other ways to contribute. Please re-review

**Comment by crweiner** [comment]
Howdy @bharatr21, thank you for your PR to add Zilliz to the Hacktoberfest Swag List! I appreciate it, especially @vimode's assist on making sure we had low/no-code options as well! Happy to get this merged!

happy cat

**Pull request merged**